dev/scripts/docker.sh

```diff
@@ -0,0 +1,269 @@
+)
+
+inference_test(){
+local TARGET=$1
+local model=$2
+if [[ ${INFERENCE_MAPPER[$model]+_} ]]; then
+    echo "${INFERENCE_MAPPER[$model]}"
+    docker exec "${TARGET}" bash -c "${INFERENCE_MAPPER["${model}"]}"
+else
+    echo "${INFERENCE_MAPPER["default"]}"
+    docker exec "${TARGET}" bash -c "llm_on_ray-serve --simple --models ${model}"
+fi
+
+echo Non-streaming query:
+docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model}"
+echo Streaming query:
+docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model} --streaming_response"
+}
+
+inference_test_deltatuner(){
+    local TARGET=$1
+    local dtuner_model=$2
+    local model=$3
+    echo "${dtuner_model}"
+    if [ "${dtuner_model}" ];then
+        docker exec "${TARGET}" bash -c "llm_on_ray-serve --config_file .github/workflows/config/mpt_deltatuner.yaml --simple"
+        docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model}"
+        docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model} --streaming_response"
+    fi
+}
+
+
+inference_test_deepspeed(){
+    local TARGET=$1
+    local model=$2
+    if [[ ${model} =~ ^(gemma-2b|gpt2|falcon-7b|starcoder|mpt-7b.*)$ ]]; then
+        echo ${model} is not supported!
+    elif [[ ! ${model} == "llama-2-7b-chat-hf-vllm" ]]; then
+        docker exec "${TARGET}" bash -c "python .github/workflows/config/update_inference_config.py --config_file llm_on_ray/inference/models/\"${model}\".yaml --output_file \"${model}\".yaml.deepspeed --deepspeed"
+        docker exec "${TARGET}" bash -c "llm_on_ray-serve --config_file \"${model}\".yaml.deepspeed --simple"
+        docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model}"
+        docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model} --streaming_response"
+    fi
+}
+
+inference_test_deepspeed_deltatuner(){
+    local TARGET=$1
+    local dtuner_model=$2
+    local model=$3
+    if [[ ${model} =~ ^(gpt2|falcon-7b|starcoder|mpt-7b.*)$ ]]; then
+        echo ${model} is not supported!
+    else
+        docker exec "${TARGET}" bash -c "llm_on_ray-serve --config_file .github/workflows/config/mpt_deltatuner_deepspeed.yaml --simple"
+        docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model}"
+        docker exec "${TARGET}" bash -c "python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/${model} --streaming_response"
+    fi
+}
+
+inference_test_restapi(){
```

Comment:
pls use test_ as suffix or test_ as prefix of al test functions.

Reply:
ok